nbs/losses.pytorch.ipynb

```diff
@@ -4361,6 +4361,46 @@
     "loss = mae(y=y, y_hat=y_hat, mask=mask)\n",
     "assert loss==(1/3), 'Should be 1/3'"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "99ada497",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "## Scaled Continuous Ranked Probability Score (sCRPS)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c66abf27",
+   "metadata": {},
+   "source": [
+    "## Unsupported losses for conformal prediction"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c960d4db",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#| export\n",
+    "UNSUPPORTED_LOSSES_CONFORMAL = (\n",
```

Comment:
I don't think we need this variable. Point losses have `loss.outputsize_multiplier = 1`, and not-point losses have `loss.outputsize_multiplier > 1`. So i'd use that as check wherever you need it. This also generalizes over future new loss functions.

Reply:
In your quick test the output should be the same as when using `MAE` (as it's the same loss). There's only a difference in the naming (when using `MAE` the output name is `NHITS`, when using `MQLoss([ ])` the output name is `NHITS-median`. But the amount of conformal predictions should be the same, only the column names should be different - I think the way you are suggesting right now.